content/articles/angular-observables/index.md

```diff
@@ -0,0 +1,313 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /angular-observables/
+title: How to Work with Observables in Angular 12
+description: In this tutorial, we discuss the concepts of Angular observables, learn what they are, how they are used in Angular applications, and finally build a sample project.
+author: owino-wendy
+date: 2022-01-03T00:00:00-17:10
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
```

Comment:
Please edit your hero image to use a capital S in the title, Subscribe to Observables.